server/src/com/openrsc/server/net/rsc/handlers/TutorialHandler.java

```diff
@@ -10,7 +10,7 @@
 
 	@Override
 	public void process(final NoPayloadStruct payload, final Player player) throws Exception {
-		if (player == null) {
+		if (player == null || !player.getWorld().getServer().getConfig().SHOW_TUTORIAL_SKIP_OPTION) {
```

Comment:
Move this check to Player.java public boolean skipTutorial().

I looked for usages of this function, and there is actually another way to skip tutorial that is not covered by placing the check here: **by using the `::skiptutorial` command.** (this command was added for usage with clients made between the release of Tutorial Island and the 2009 reopening)

It is also good practice to place it in skipTutorial() since other sanity check logic is there.

Reply:
@Hubcapp done. Btw, that `::skiptutorial` command doesn't work for me, even in `develop`.

Reply:
@Hubcapp can you take a look when you have a chance?

Reply:
approved, thanks for your work